packages/src/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Internal dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
```

Comment:
In order to make it uniform with other blocks bellow, please add a space after this block.

Reply:
The suggestion was implemented.